src/Tools/GetDocumentInsider/src/Commands/GetDocumentCommand.cs

```diff
@@ -5,7 +5,7 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
-#if NETCOREAPP2_1
+#if NETCOREAPP2_1 || NET7_0_OR_GREATER
```

Comment:
Is it really anything netcoreapp2.1 and above? I think `NETCOREAPP` covers that.